<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamepadSettings</class>
 <widget class="QWidget" name="GamepadSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>530</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>538</width>
        <height>530</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
       <item>
        <widget class="QGroupBox" name="groupButtons">
         <property name="title">
          <string>按钮设置</string>
         </property>
         <layout class="QFormLayout" name="buttonsLayout"/>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupAxes">
           <property name="title">
            <string>摇杆设置</string>
           </property>
           <layout class="QFormLayout" name="axesLayout"/>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupTBD">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string/>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupLeft">
              <property name="title">
               <string>左摇杆</string>
              </property>
              <layout class="QFormLayout" name="formLayout_4">
               <item row="0" column="0">
                <widget class="QLabel" name="titleLeftX">
                 <property name="text">
                  <string>水平</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="axisLeftX"/>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="reverseLeftX">
                 <property name="text">
                  <string>反转</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QProgressBar" name="statusLeftX">
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="format">
                  <string>%v%</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="titleLeftY">
                 <property name="text">
                  <string>垂直</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="axisLeftY"/>
               </item>
               <item row="4" column="1">
                <widget class="QCheckBox" name="reverseLeftY">
                 <property name="text">
                  <string>反转</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QProgressBar" name="statusLeftY">
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="format">
                  <string>%v%</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QGroupBox" name="groupRight">
              <property name="title">
               <string>右摇杆</string>
              </property>
              <layout class="QFormLayout" name="formLayout_2">
               <item row="0" column="0">
                <widget class="QLabel" name="titleRightX">
                 <property name="text">
                  <string>水平</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="axisRightX"/>
               </item>
               <item row="1" column="1">
                <widget class="QCheckBox" name="reverseRightX">
                 <property name="text">
                  <string>反转</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QProgressBar" name="statusRightX">
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="format">
                  <string>%v%</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="titleRightY">
                 <property name="text">
                  <string>垂直</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="axisRightY"/>
               </item>
               <item row="4" column="1">
                <widget class="QCheckBox" name="reverseRightY">
                 <property name="text">
                  <string>反转</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QProgressBar" name="statusRightY">
                 <property name="minimum">
                  <number>-100</number>
                 </property>
                 <property name="value">
                  <number>0</number>
                 </property>
                 <property name="format">
                  <string>%v%</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="enableTitle_1">
                <property name="text">
                 <string>按住</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="enable1"/>
              </item>
              <item>
               <widget class="QLabel" name="enable1Status">
                <property name="pixmap">
                 <pixmap resource="../../resources/icons/Fatcow.qrc">:/fatcow/16x16/bullet_green.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="enableTitle_2">
                <property name="text">
                 <string>和</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="enable2"/>
              </item>
              <item>
               <widget class="QLabel" name="enable2Status">
                <property name="pixmap">
                 <pixmap resource="../../resources/icons/Fatcow.qrc">:/fatcow/16x16/bullet_green.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="enableTitle_3">
                <property name="text">
                 <string>时允许运动</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="3" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="stopTitle_1">
                <property name="text">
                 <string>按</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="stopB"/>
              </item>
              <item>
               <widget class="QLabel" name="stopStatus">
                <property name="pixmap">
                 <pixmap resource="../../resources/icons/Fatcow.qrc">:/fatcow/16x16/bullet_green.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="stopTitle_2">
                <property name="text">
                 <string>时停止所有运动</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="4" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="0" colspan="2">
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
              <item>
               <widget class="QLabel" name="gamepadTitle">
                <property name="text">
                 <string>手柄</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="gamepad"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/icons/Fatcow.qrc"/>
 </resources>
 <connections/>
</ui>
